content/news/2023-07-07-bevy-0.11/index.md

```diff
@@ -0,0 +1,639 @@
+        base_color: Color::WHITE,
+        ..default()
+    }),
+    ..default()
+});
+```
+
+![A 3D desert scene with two flat white walls and a pebble path winding between them](parallax_mapping_none.jpg)
+
+Of course, it's just a flat white box, we didn't add any texture.
+So let's add a normal map:
+
+```rust
+normal_map_texture: Some(assets.load("normal_map.png")),
+```
+
+![The same scene with normal maps](parallax_mapping_normals.jpg)
+
+This is much better. The shading changes according to the light direction too!
+However, the specular highlights on the corner are overbearing, almost noisy.
+
+Let's see how a depth map can help:
+
+```rust
+depth_map: Some(assets.load("depth_map.png")),
+```
+
+![The same scene with a depth texture](parallax_mapping_depth.jpg)
+
+We eliminated the noise! There is also that sweet 3D feel reminiscent of
+90's games pre-rendered cinematic sequences.
+
+So what's going on, why does parallax mapping eliminate the ugly specular
+lights on the wall?
+
+This is because parallax mapping insets the ridges between bricks, so that they
+are occluded by the bricks themselves.
+
+![Illustration of the previous paragraph](ridge-light-view-1.svg)
+
+Since normal maps do not "move" the shaded areas, merely shade them
+differently, we get those awkward specular highlights. With parallax mapping,
+they are gone.
+
+![A montage of the three preceding images, contrasting each effect](parallax_mapping_compare.jpg)
+
+Parallax mapping in Bevy is still very limited. The most painful aspect is that
+it is not a standard glTF feature, meaning that the depth texture needs to be
+programmatically added to materials if they came from a GLTF file.
+
+Additionally, parallax mapping is incompatible with the temporal antialiasing
+shader, doesn't work well on curved surfaces, and doesn't affect object's
+silhouettes.
+
+However, those are not fundamental limitations of parallax mapping, and may be
+fixed in the future.
+
+## WebGPU Support
+
+## Deref Derive Attribute
+
+<div class="release-feature-authors">authors: @MrGVSV</div>
+
+Bevy code tends to make heavy use of the [newtype](https://doc.rust-lang.org/rust-by-example/generics/new_types.html) pattern,
+which is why we have dedicated derives for [`Deref`](https://docs.rs/bevy/latest/bevy/prelude/derive.Deref.html) and [`DerefMut`](https://docs.rs/bevy/latest/bevy/prelude/derive.DerefMut.html).
+
+This previously only worked for structs with a single field:
+
+```rust
+#[derive(Resource, Deref, DerefMut)]
+struct Score(i32);
+```
+
+For 0.11, we've improved these derives by adding the `#[deref]` attribute, which allows them to be used on structs with multiple fields.
+This makes working with generic newtypes much easier:
+
+```rust
+#[derive(Component, Deref, DerefMut)]
+struct Health<T: Character> {
+    #[deref] // <- use the `health` field as the `Deref` and `DerefMut` target
+    health: u16,
+    _character_type: PhantomData<T>,
+}
+```
```

Comment:
Looks like this section is misplaced inside another section. I'm also guessing it's meant to be much lower in the post.